### PR TITLE
Issue-2-view-entries

### PR DESCRIPTION
**Pull request by ChorongShin**
https://user-images.githubusercontent.com/91721272/171109941-5b503795-51a1-436e-a3b9-b3b6bf155e70.mov

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -26,4 +30,64 @@ $form.addEventListener('submit', function (event) {
   data.entries.unshift(entry);
   $placeholder.src = 'images/placeholder-image-square.jpg';
   $form.reset();
+
+  window.location.reload();
```
Comment (BrettAlbright) [inline js/main.js]: you cannot reload the application during run time to update the DOM. Instead, you want to read through the hints about using the `prepend` method to add the new entry to the DOM when the form is submitted. Make sure to reach out with any questions!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
update needed.

**Comment by ChorongShin** [comment]
https://user-images.githubusercontent.com/91721272/171252344-2fdd2467-ebe9-494f-862f-d8217d227d1c.mov

**Comment by ChorongShin** [comment]
https://user-images.githubusercontent.com/91721272/171306543-98217afa-444b-4328-b241-abe077623f5a.mov

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**